Plan/sponge/src/main/java/com/djrapitops/plan/PlanSponge.java

```diff
@@ -199,21 +217,14 @@ public ColorScheme getColorScheme() {
 
     @Override
     public void registerCommand(Subcommand command) {
-        if (command == null) {
-            logger.warn("Attempted to register a null command!");
-            return;
-        }
-        for (String name : command.getAliases()) {
-            CommandManager commandManager = Sponge.getCommandManager();
-
-            CommandMapping registered = commands.get(name);
-            if (registered != null) {
-                commandManager.removeMapping(registered);
-            }
+        // NOOP: Sponge commands are registered via the event below
+    }
 
-            Optional<CommandMapping> register = commandManager.register(this, new SpongeCommand(runnableFactory, system.getErrorLogger(), command), name);
-            register.ifPresent(commandMapping -> commands.put(name, commandMapping));
-        }
+    @Listener
+    public void onRegisterCommand(RegisterCommandEvent<Command.Raw> event) {
```

Comment:
Does the command get registered with the new system when plugin reloads or does it still use the previously initialized system that gets deleted?

Reply:
https://github.com/plan-player-analytics/Plan/commit/002ab1b11337803a27f7253ae41acf9823bb3fda